db/migrate/20221123110221_add_user_ref_to_post.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToPost < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
+  end
```

Comment:
- [ ] Please rename the `user` foreign key column to `author` instead, as per the following [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png).